README.md

```diff
@@ -167,18 +167,18 @@ this functionality is currently not implemented.
 
 If you want to take a photo,
 press the `Capture Image` button
-and the software will save an image in the 
-`.jpeg` format in the location of your choice. 
+and the software will save an image in the
+`.jpeg` format in the location of your choice.
 Other formats are currently not supported,
-but will be in the future. 
-You can also easily change the default 
+but will be in the future.
```

Comment:
I could add other formats, should I?

Reply:
Yes, but let's not blow this PR up further :) Let's do a separate PR there